sdk/securitycenter/Azure.ResourceManager.SecurityCenter/api/Azure.ResourceManager.SecurityCenter.netstandard2.0.cs

```diff
@@ -1,5374 +0,0 @@
-namespace Azure.ResourceManager.SecurityCenter
```

Comment:
why is this removed?
And all removed other API files not in the current changing RP.

Reply:
fixed